.ci.yaml

```diff
@@ -3237,7 +3237,7 @@ targets:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:98"},
+          {"dependency": "chrome_and_driver", "version": "version:98.1"},
```

Comment:
Can we update all the references of version:98 to version:98.1

Reply:
Done